### ISSUE TITLE
docs(google): access blocked

### ISSUE THREAD

**Issue by josephmcasey** [open-telemetry/opentelemetry-specification#3030]
Documentation such as the links below are unable to be viewed by individuals working for companies using Google as an enterprise offering. I believe the primary security concern is that employees will mistake public documents as internal and document company secrets on public documents. 🤷 

https://github.com/open-telemetry/opentelemetry-specification/blob/5b6d22512ef72214f7cbd52747a1fbfe49f8121f/specification/logs/data-model.md?plain=1#L1294

Error:

> Can't access item
> Your organization's sharing policy prevents you from accessing this item

**Comment by jack-berg** [comment]
Yikes that's a silly policy

**Comment by arminru** [comment]
Would you be able to access it from a separate browser or a private window where you're not logged in to that enterprise account?

**Comment by josephmcasey** [comment]
> Would you be able to access it from a separate browser or a private window where you're not logged in to that enterprise account?

Unfortunately not due to the login to other accounts being blocked.

> As an administrator, you might want to prevent users from signing in to Google services using any accounts other than those you provided them with. For example, you might not want users within your corporate network to use their personal Gmail accounts or a managed Google Account from another domain.
>
> Note: When you block access to consumer accounts, users might see the following error message: "This account is not allowed to sign in within this network".
_[Source](https://support.google.com/a/answer/1668854?hl=en)_

**Comment by jsuereth** [comment]
I'm not sure how to resolve this.  Are you looking just for historical information on the discussion that happened or do you want continued active views of the document?

**Comment by ocelotl** [comment]
I asked Claude to review this issue and Claude suggests it be closed with this explanation:

> Not an actionable spec change: an external enterprise Google sharing policy blocks the reporter from viewing a linked design doc. Labeled triaged-needmoreinfo; jsuereth's Jan 2023 question ('historical info or continued active views?') was never answered and the thread has been dead since. jack-berg/maintainers signaled no clear path ('not sure how to resolve this').

To guard against a mistaken suggestion, Claude then independently re-verified this conclusion:

> Closeable, but not stale-abandoned: the inaccessible docs.google.com links the reporter complained about have since been removed from specification/logs/data-model.md (References section on main now only lists the GitHub OTEP 0097 link), so the underlying ask is resolved, not merely abandoned.

Claude evaluated this issue with medium confidence that it can be closed.

Claude has classified this issue as resolved.

The intention of this comment is to make it easier for a human being to review this issue before making a decision.

**Comment by ocelotl** [comment]
I disagree with Claude marking this resolved because there are still links to docs.google in the spec. Nevertheless, I don't see much point in keeping this open. Nobody else has reported this problem, the person who submitted this issue has also closed their corresponding Github account, so it is not possible to continue a conversation with this person. I think this can be closed.
